apps/admin_api/lib/admin_api/v1/router.ex

```diff
@@ -129,11 +129,14 @@ defmodule AdminAPI.V1.Router do
     # API Access endpoints
     post("/access_key.all", KeyController, :all)
     post("/access_key.create", KeyController, :create)
+    post("/access_key.enable_or_disable", KeyController, :enable_or_disable)
+    # Deprecated
```

Comment:
This is ambiguous whether both update and delete are deprecated.

Maybe move the deprecated endpoint to the end of this section?

```ex
    ..
    post("/access_key.enable_or_disable", KeyController, :enable_or_disable)
    post("/access_key.delete", KeyController, :delete)

    # Deprecated
    post("/access_key.update", KeyController, :update)

    # API Key endpoints
    post("/api_key.all", APIKeyController, :all)
```